src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,119 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
```

Comment:
use display: flex for container and specify needed position for child elements, instead of setting margin-right or bottom for each card